### PR TITLE
Move loading spinner to its own file

### PR DESCRIPTION
**Pull request by swashcap**
# Problem

The `#spinner` and its implementation are closely coupled to the `App` component.

referenced issue(s): _n/a_

# Solution

Refactor by moving the spinner to a new `ActivityIndicator` component. Move the styles to clean things up.

# Testing

Try logging in:

it-still-works

### CODE REVIEW

File: packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+
+.activity-indicator {
```
Comment (swashcap) [inline packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.css:2]: Copied these rules directly out of _app.scss_. Should maybe clean them up, too. They work.

File: packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import './activity-indicator.css';
+
+export default class ActivityIndicator extends React.Component {
```
Comment (swashcap) [inline packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.jsx:7]: My general thinking with this PR is that it's easy to swap our implementation for something else (like [Material Design's “Progress Indicators”](https://material.io/design/components/progress-indicators.html)). It also cleans up the `App` component.

File: packages/coinstac-ui/app/render/components/app.jsx [hunk 3]
```diff
@@ -34,16 +35,11 @@ class App extends Component { // eslint-disable-line react/prefer-stateless-func
   }
 
   render() {
-    const { children, loading: { isLoading, wip }, notifications } = this.props;
```
Comment (swashcap) [inline packages/coinstac-ui/app/render/components/app.jsx:37]: `wip` wasn't used. Wonder why the linter didn't catch?

File: packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.jsx (hunk 2 above)
Comment (rssk) [reply packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.jsx:7]: 👍

**Review by rssk — APPROVED** [review]

**Pull request merged**